bench/db_bench.ml

```diff
@@ -14,13 +14,13 @@ let () = Random.init seed
 let key_size = 44
 
 (* the hash_size in pack_index*)
-let hash_size = 60
+let hash_size = 30
 
 let value_size = 13
 
 let nb_entries = 1_000_000
 
-(* in the tezos use case, log_size is proportional to the nb_entries *)
+(* in tezos the log_size is 500_000  *)
 let log_size = nb_entries / 5
```

Comment:
can't we use `log_size = 500_000` here too?

Reply:
We can, but I think we are a bit lying to ourselves. The larger the log size the better we do for these small benchmarks. But then what we measure here will no reflect the performance for a bigger number of entries. 
Maybe it's fine, as probably lmdb's perf degrades also for large datasets.

Reply:
we are not lying, we are just using the same cache parameters as used in production. We should make sure we are not introducing bias so for instance we should make sure that the read tests are done on "cold" boot, e.g. with empty caches. But it's perfectly fine to use and have the right write caches. lmdb is doing the same internally!